### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by chatziko** [ronengr/hass_nuki_bt#31] [General]
Quick note to let you know that I [submitted brand icons](https://github.com/home-assistant/brands/pull/4933) for hass_nuki_bt, so now we get nice icons in the devices page :smile:

**Comment by ronengr** [comment]
Thanks!

is there anything i have to do on my side for this to work?

**Reply by chatziko** [reply]
No, it's already working on my system, try to refresh a few times.

![Screenshot from 2023-11-27 21-43-18](https://github.com/ronengr/hass_nuki_bt/assets/362089/3a36727e-1289-4c1e-a1c7-c6e4ad9c6feb)
